src/services/phoneNumber.service.ts

```diff
@@ -1,7 +1,11 @@
 import PhoneNumber, { IPhoneNumber } from '../models/phoneNumber.model';
 
 export const getPhoneNumberByCountry = async (countryId: string): Promise<IPhoneNumber[]> => {
-  const phoneNumbers: IPhoneNumber[] = await PhoneNumber.find({country_id: countryId, is_active: true});
+  const phoneNumbers: IPhoneNumber[] = await PhoneNumber.find({ country_id: countryId, is_active: true });
 
   return phoneNumbers;
-}
+};
+
+export const getInactivePhoneNumber = async () => {
+  return await PhoneNumber.find({ is_active: false }).populate({ path: 'country_id', model: 'Country' });
```

Comment:
Có thể viết thế này luôn k c?
```suggestion
  return await PhoneNumber.find({ is_active: false }).populate('country_id');
```